### DISCUSSION TITLE
Program freezing after a certain amount of time

### DISCUSSION THREAD

**Discussion by steevenakintilo** [tweepy/tweepy#1669] [Questions]
Hello I've made a twiiter bot who is working well but after a certain amount of time like 10 minutes the bot freeze and doesn't work anymore and I've to restart I don't understand why.
My code:
https://pastebin.com/phb7VY4j

**Comment by Harmon758** [comment]
This is too much code to debug for you, with a lot of unnecessary nesting and repetition, but it's likely not an issue with Tweepy.
Can you provide a reproducible example (MCVE/SSCCE) or be more specific about your issue?

**Reply by steevenakintilo** [reply]
I fixed the bug yesterday It wasn't really a tweepy issue but I realized that when My program was freezing it was just because my bot wasn't used that's all.
